frontend/src/components/editProject/EditCollaboratingOrganizations.js

```diff
@@ -0,0 +1,53 @@
+import { List } from "@material-ui/core";
+import React from "react";
+
+import MiniOrganizationPreview from "../organization/MiniOrganizationPreview";
+import AutoCompleteSearchBar from "../search/AutoCompleteSearchBar";
+
+export default function EditCollaboratingOrganizations({
+  texts,
+  baseUrl,
+  searchBarClass,
+  handleAddCollaboratingOrg,
+  organizationPreviewsContainerClass,
+  organizationPreviewClass,
+  selectedCollaboratingOrgs,
+  handleDeleteCollaboratingOrg,
+}) {
+  const renderSearchOption = (option) => {
+    return <React.Fragment>{option.name}</React.Fragment>;
```

Comment:
Nice - looks like we have a mix of `<React.Fragment>` usages alongside `<>` throughout the codebase. Do you have a preference or POV on which to use consistently?

Reply:
I usually just user `<>`, in this case I copied the function from elsewhere in the codebase which uses `<React.Fragment>`. Changed to  `<>` now